compiler/src/latex_printer.ml

```diff
@@ -139,7 +168,7 @@ let pp_aligned =
 let rec pp_simple_attribute fmt a =
   match L.unloc a with
   | Aint i -> Z.pp_print fmt i
-  | Aid s | Astring s -> pannot fmt s
+  | Aid s | Astring s -> pannot fmt (Format.asprintf "%a" pp_string s)
```

Comment:
The old code should be correct for the `Aid` case.

Reply:
Don't we allow underscores in identifiers?

Reply:
Underscores are letters (catcode 11).

Reply:
done